src/pages/resultsView/mutualExclusivity/MutualExclusivityTab.tsx

```diff
@@ -76,11 +76,21 @@ export default class MutualExclusivityTab extends React.Component<IMutualExclusi
         this.significantPairsFilter = !this.significantPairsFilter;
     }
 
+    @computed get error() {
```

Comment:
can this have a more descriptive name?  e.g. XYZBrandErrorText

Reply:
sure